piksi_tools/console/console.py

```diff
@@ -312,7 +317,8 @@ def __init__(self):
 
 with serial_link.get_driver(args.ftdi, port, baud) as driver:
   with sbp.client.handler.Handler(driver.read, driver.write, args.verbose) as link:
-    with serial_link.get_logger(args.log, serial_link.LOG_FILENAME) as logger:
+    logfilename = log_file_prefix + "_" + time.strftime("%Y%m%d-%H%M%S.log.json")
```

Comment:
This should be probably be `logfilename = log_file_prefix + "_" + `serial_link.LOG_FILENAME)`